tests/test_core/test_new_alignment.py

```diff
@@ -362,6 +363,19 @@ def test_seqs_data_to_alphabet_invalid():
         _ = seqs.to_alphabet(DNA)
 
 
+@pytest.mark.parametrize("reverse", (False, True))
+def test_seqs_data_round_trip(reverse, alpha):
+    seqs_data = new_aln.SeqsData(
+        data={"seq1": "ACGG", "seq2": "CGCA", "seq3": "CCG-"}, alphabet=alpha
+    )
+    seqs_data = seqs_data.reverse_seqs() if reverse else seqs_data
+
+    rd = seqs_data.to_rich_dict()
+    got = deserialise_object(rd)
+    assert isinstance(got, new_aln.SeqsData)
+    assert got.to_rich_dict() == seqs_data.to_rich_dict()
```

Comment:
this is just exercising the reverse method. I think you should check the value for at least one sequence is correct against expectation when you use `str()` on `get_seq_view()`.

Reply:
good point!